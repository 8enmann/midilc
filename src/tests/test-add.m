/**
   Test add
   Tests adding a duration to a note 
   @author Fred
   */

main() {
  Note a;
  Sequence b;
  Chord c;
  b = new_sequence();
  a = A3q + 4;
  c = new_chord(a);
  b = b + c;
  b = b + b;
  play(b);
}
