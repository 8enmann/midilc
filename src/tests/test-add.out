0,8,57
8,8,57

